automation/properties/inclusiveness-analyser.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Inclusiveness Analyzer",
+    "creator": "Microsoft",
```

Comment:
@cblomart what is the support policy from Microsoft on this?

Reply:
hi @bishal-pdMSFT this is an opensource project published under the microsoft org: https://github.com/microsoft/InclusivenessAnalyzer

It is published under MIT license and is uniquely supported by the maintaining team via github issues.
We are in the process of reviewing the SUPPORT.md file to provide more informations.